pages/firebase-firsttry.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { collection, onSnapshot } from 'firebase/firestore';
+import Link from 'next/link';
+
+import { db } from '../firebase/initFirebase';
+
+const FirebaseFirst = () => {
+  const [articleList, setArticleList] = useState([]);
+  const articlesCollectionRef = collection(db, 'articles');
```

Comment:
Could we move this next to the `onSnapshot` function?

Reply:
?